src/index/IndexImpl.cpp

```diff
@@ -1431,9 +1431,17 @@ bool IndexImpl::getId(const string& element, Id* id) const {
 }
 
 // ___________________________________________________________________________
-std::pair<Id, Id> IndexImpl::prefix_range(const std::string& prefix) const {
+std::pair<Id, Id> IndexImpl::prefix_range_internal_vocabulary(
+    const std::string& prefix) const {
   // TODO<joka921> Do we need prefix ranges for numbers?
-  auto [begin, end] = vocab_.prefix_range(prefix);
+  auto [begin, end] = vocab_.prefix_range_internal_vocabulary(prefix);
+  return {Id::makeFromVocabIndex(begin), Id::makeFromVocabIndex(end)};
+}
+
+// ___________________________________________________________________________
+std::pair<Id, Id> IndexImpl::prefix_range_external_vocabulary(
```

Comment:
Actually (What I just realized) our convention for function names is `prefixRangeExternalVocab`.
We sometimes have slight exceptions for functions that have the same name as in similar standard library containers to be able to write generic code, but that doesn't apply here.